src/config/dynamic-theme-fixes.config

```diff
@@ -5221,7 +5221,10 @@ div[role="button"][aria-disabled="true"] > div {
     filter: invert(93.7%) hue-rotate(180deg) contrast(90.6%) !important 
 }
 :root {
-    --dt-secondary-container: ${#f1f3f4} !important;
+    --dt-secondary-container: #2f3335;
+}
+button[guidedhelpid="new_menu_button"]{ 
+    background-color: var(--dt-secondary-container)!important;
```

Comment:
What's the current background-color? We normally don't add CSS to then force it to use the original variables(we just fix the variable itself then).

Reply:
Without this fix the button have the same background-color of the whole page.
The bar on trash folder without this fix have a black background.

i'm trying to fix using the ${...} anyway. for now i've delete this part of modification